go.mod

```diff
@@ -76,10 +76,10 @@ require (
 	github.com/rivo/tview v0.0.0-20220709181631-73bf2902b59a
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/rogpeppe/go-internal v1.9.0
-	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/cobra v1.4.0
```

Comment:
Did you add `-u` when running `go get`  @suwliang3 . That would bump the all go modules versions although the backport doesn't need them.

Reply:
I didn't run `go get`. I run `go mod tidy`.

Reply:
Why `go mod tidy`? I think `go get github.com/kubevela/pkg@263dc191bf51` is enough. Can you include the essential changes only?

Reply:
Sorry @suwliang3 I just checked the build process and I found it surely have issues with just `go get github.com/kubevela/pkg@263dc191bf51`. I have just restart the apiserver test which fails.